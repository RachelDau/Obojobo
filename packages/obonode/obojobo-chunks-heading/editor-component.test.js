import React from 'react'
import renderer from 'react-test-renderer'

import Heading from './editor-component'

jest.mock(
	'obojobo-document-engine/src/scripts/oboeditor/components/node/with-slate-wrapper',
	() => item => item
)
jest.mock(
	'obojobo-document-engine/src/scripts/oboeditor/components/node/editor-component',
	() => props => <div>{props.children}</div>
)

describe('Heading Editor Node', () => {
	test('Heading component', () => {
		const component = renderer.create(<Heading element={{ content: {} }} />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
