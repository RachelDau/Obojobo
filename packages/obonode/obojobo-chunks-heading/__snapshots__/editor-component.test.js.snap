// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Editor Node Heading component 1`] = `
<div>
  <div
    className="text-chunk obojobo-draft--chunks--heading pad"
  >
    <h1>
      <span
        className="text align-undefined"
      />
    </h1>
  </div>
</div>
`;
